.pre-commit-config.yaml

```
@@ -29,6 +29,14 @@ repos:
       - id: requirements-txt-fixer
       - id: trailing-whitespace
 
+  # Golang hooks
+  - repo: https://github.com/dnephin/pre-commit-golang
+    rev: v0.5.0
+    hooks:
+      - id: go-fmt
+      - id: no-go-testing
+      - id: go-unit-tests
```

Sort hooks alphabetically:
```suggestion
      - id: go-fmt
      - id: go-unit-tests
      - id: no-go-testing
```

---

resolved with [e3c2e82](https://github.com/cisagov/con-pca-tasks/pull/1/commits/e3c2e82955ab02e64b5d857403dcf2d4c18bedd8)